sites/public/src/components/listing/ListingView.tsx

```diff
@@ -182,8 +182,15 @@ export const ListingView = (props: ListingProps) => {
       !listingPref.multiselectQuestions.hideFromListing
   )
 
-  const getPreferenceData = () => {
-    return listingPreferences.map((listingPref, index) => {
+  const listingPrograms = listing?.listingMultiselectQuestions.filter(
+    (listingProgram) =>
+      listingProgram.multiselectQuestions.applicationSection ===
+        MultiselectQuestionsApplicationSectionEnum.programs &&
+      !listingProgram.multiselectQuestions.hideFromListing
+  )
+
+  const getPreferenceData = (forPrograms = false) => {
```

Comment:
Rather than a fxn called `getPreferenceData` that takes a `programs` argument, can we make the fxn named something more generic, and pass in `MultiselectQuestionsApplicationSectionEnum`?